PWGCF/FemtoWorld/Core/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+# All rights not expressly granted are reserved.
+#
+# This software is distributed under the terms of the GNU General Public
+# License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
```

Comment:
Have in mind that you don't necessarily need to only have headers files in your `Core` directory. 
You could also have `.cxx` files when needed and in that case you should instruct the compiler to create a library that you will link with your table producer tasks or with your task as needed
Check https://github.com/AliceO2Group/O2Physics/blob/master/PWGCF/Core/CMakeLists.txt for inspiration